tensorzero-core/src/providers/anthropic.rs

```diff
@@ -580,12 +579,6 @@ impl<'a> AnthropicMessageContent<'a> {
                 _ => {
                     // Otherwise, fetch the file, encode it as base64, and send it to Anthropic
 
-                    warn_cannot_forward_url_if_missing_mime_type(
```

Comment:
Can you add this back, since we really do need a mime type to determine how to pass a url to anthropic